### ISSUE TITLE
Upload package to pypi

### ISSUE THREAD

**Issue by PabloCastellano** [gdamdam/saitan#4]
It would be very comfortable having this tool available in Pypi so that it can be installed using `pip install`

**Comment by gdamdam** [comment]
yes, but the project require more work to be ready to be on pypi.... 
and I haven't time ATM
